### PR TITLE
KSE3-565 Universal Animated Loader 0.9

### PR DESCRIPTION
**Pull request by mariokleen**
image
image

### CODE REVIEW

File: libs/atomic-elements/src/lib/components/Loader/Loader.scss [hunk 1]
```diff
@@ -0,0 +1,54 @@
+.loader {
+  width: 100%;
+  height: 100%;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  color: var(--grey_mid_0x);
```
Comment (sebaskleeen) [inline libs/atomic-elements/src/lib/components/Loader/Loader.scss:7]: lets use something from the palette

File: libs/atomic-elements/src/lib/components/Loader/Loader.scss [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+	background: linear-gradient(-90deg, #FFFFFF, #ececec, #FFFFFF, #ececec);
```
Comment (sebaskleeen) [inline libs/atomic-elements/src/lib/components/Loader/Loader.scss:8]: Same here, colors from the palette, does this look good with dark backgrounds?

File: libs/atomic-elements/src/lib/components/Loader/Loader.scss [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+	background-size: 400% 400%;
+  animation: gradient 16s ease infinite;
+}
+
+.message {
```
Comment (sebaskleeen) [inline libs/atomic-elements/src/lib/components/Loader/Loader.scss:13]: Anything in our app with the class `.message` or `.loader` will collide with this classes. At the very least we should scope this classes in some class like `.ks-loader`. But ideally we can use `makeStyles` like in our other components ❤️ if we are gonna do everything in a material like way, we should slowly stop using `scss`

**Comment by mariokleen** [comment]
After talking with Josh, we'll use material loaders for now. This will be used on future versions of loader, so let's keep the branch

**Pull request closed without merging**